### PR TITLE
Added set-udclipboard. 

### PR DESCRIPTION
**Pull request by BoSen29**
Might update at later stage to allow copying the specified clientside ID to the clipboard.

### CODE REVIEW

**Comment by BoSen29** [comment]
Fixes #1190

**Comment by adamdriscoll** [comment]
This pull request has been mentioned on **PowerShell Universal Dashboard Forums**. There might be relevant details there:

https://forums.universaldashboard.io/t/copy-string-to-clipboard-via-udbutton/1328/6

**Comment by adamdriscoll** [comment]
One small nit but this looks great!

File: src/UniversalDashboard/Cmdlets/Set-UDClipboard.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+namespace UniversalDashboard.Cmdlets
+{
+    [Cmdlet(VerbsCommon.Set, "UDClipboard")]
+    public class SetUDClipboardCommand : PSCmdlet
+    {
+		//private readonly Logger Log = LogManager.GetLogger(nameof(InvokeRedirectCommand));
+
+        [Parameter(Mandatory = true)]
+		public string data { get; set; }
```
[the remark this answers is not in the document]
Comment (BoSen29) [reply src/UniversalDashboard/Cmdlets/Set-UDClipboard.cs]: Fixed :)

**Comment by adamdriscoll** [comment]
Great! Thanks!

**Pull request merged**